flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/utils/TestData.scala

```diff
@@ -473,7 +473,7 @@ object TestData {
       3.14,
       "EFG",
       localDate("2017-05-20"),
-      localTime("09:45:78"),
```

Comment:
I had to change the seconds to a valid range (0-59). Otherwise, the new validation logic in parseTIme would return a null, and the test `OverAggregateITCase` would fail.